met-api/src/met_api/resources/widget_documents.py

```diff
@@ -54,3 +56,35 @@ def post(widget_id):
             return ActionResult.success(result=WidgetDocumentsSchema().dump(document))
         except BusinessException as err:
             return ActionResult.error(str(err))
+
+
+@cors_preflight('PATCH, DELETE')
+@API.route('/<document_id>')
+class WidgetDocumentsChanges(Resource):
+    """Resource for managing documents with documents widget."""
+
+    @staticmethod
+    @cross_origin(origins=allowedorigins())
+    @auth.require
+    def patch(widget_id, document_id):
+        """Update saved documents."""
+        request_json = request.get_json()
+        user_id = TokenInfo.get_id()
+        request_json['modified_by_id'] = user_id
+        request_json['updated_date'] = datetime.utcnow()
+        try:
```

Comment:
since widgt_doc inherits base model , this values will be populated automatically i think..so you can remove it